workflow-manager/kubernetes/kubernetes.go

```diff
@@ -3,6 +3,11 @@ package kubernetes
 
 import (
 	"fmt"
+	"net"
+	"os"
+	"time"
+
+	"gopkg.in/retry.v1"
```

Comment:
Good find, I wish we had found this package earlier! Would you mind deleting the (unused) `workflow-manager/retry` package in this PR?

Reply:
yes, can do.